website/content/en/status/report-2023-01-2023-03/pot.adoc

```diff
@@ -0,0 +1,20 @@
+=== Containers and FreeBSD: Pot, Potluck and Potman
+
+Links: +
+link:https://github.com/bsdpot[Pot organization on GitHub] URL: link:https://github.com/bsdpot[https://github.com/bsdpot]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Bretton Vine (Potluck) <bv@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/commits/master[pot] received a number of minor fixes but no new version has been released yet.
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases Nomad.
+
+All Potluck images have been rebuilt to include the latest FreeBSD security advisories, a new link:https://github.com/bsdpot/potluck/tree/master/smokeping[Smokeping network latency monitoring image has been added] and again a lot of work went into the link:https://github.com/bsdpot/potluck/tree/master/jitsi-meet[Jitsi] image which unfortunately still seems to have some reliability issues.
+
+Also, two new blog posts are available showing how easy it should be to use Potluck images, link:https://honeyguide.eu/posts/minio-beast-nextcloud/[one explaining how to set up Nextcloud with Minio as object storage and Prometheus for monitoring], link:https://honeyguide.eu/posts/openldap-matrix-blog-post/[one showing how to run your own Matrix Synapse server using OpenLDAP for access management].
```

Comment:
"how easy it should be"? Or "how easy it is"? To me, the former hints that it might not be as easy as it ought to.